src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -2661,6 +2661,507 @@ Note: `aparse()` is case-insensitive.
+
+    <Callout variant="tip">
+      If the string found does not match the given pattern, it will return `null`. If you happen to have datetime strings in a variety of patterns, you can coalesce results by leveraging `OR` to cascade until one of the values is non-null. You can also use optional pattern segments. The default pattern uses square brackets to make the milliseconds and zone-offset parts optional.
+    </Callout>
+
+    <CollapserGroup>
+      <Collapser id="partial-datetime-interpretation" title="Partial datetime interpretation when parsing datetime strings">
+        <table>
+          <thead>
+          <tr>
+            <th>
+              Scenario
+            </th>
+            <th>
+              Detail
+            </th>
+            <th>
+              Sample pattern
+            </th>
+            <th>
+              Sample datetime
+            </th>
+            <th>
+              Resolves to
+            </th>
+          </tr>
+          </thead>
+
+          <tbody>
+          <tr>
+            <td>Missing time zone</td>
+            <td>Will use timezone argument, or WITH TIMEZONE value. Defaults to UTC.</td>
+            <td>yyyy-MM-dd HH:mm:ss.SSS</td>
+            <td>2023-10-18 15:27:03.123</td>
+            <td>2023-10-18T15:27:03.123Z</td>
+          </tr>
+          <tr>
+            <td>Missing datetime field</td>
+            <td>
+              Any missing time field will be replaced with 0.<br/>
+              Any missing date field will be replaced with 1.<br/>
+              If a field is present, all higher precedence fields must also be present.<br/>
+              Note that day of year and quarter of year patterns are supported.
+            </td>
+            <td>
+              yyyy-MM-dd HH:mm:ss<br/>
+              yyyy-MM-dd HH:mm<br/>
+              yyyy-MM-dd HH<br/>
+              yyyy-MM-dd<br/>
+              yyyy 'day' D<br/>
+              yyyy-MM<br/>
+              yyyy qqq<br/>
+              yyyy
+            </td>
+            <td>
+              2023-10-18 15:27:03<br/>
+              2023-10-18 15:27<br/>
+              2023-10-18 15<br/>
+              2023-10-18<br/>
+              2023 day 291<br/>
+              2023-10<br/>
+              2023 Q4<br/>
+              2023
+            </td>
+            <td>
+              2023-10-18T15:27:03.000Z<br/>
+              2023-10-18T15:27:00.000Z<br/>
+              2023-10-18T15:00:00.000Z<br/>
+              2023-10-18T00:00:00.000Z<br/>
+              2023-10-18T00:00:00.000Z<br/>
+              2023-10-01T00:00:00.000Z<br/>
+              2023-10-01T00:00:00.000Z<br/>
+              2023-01-01T00:00:00.000Z<br/>
+            </td>
+          </tr>
+          <tr>
+            <td>Time only</td>
+            <td>
+              If a time pattern is used without a date, the millisecond of the day will be calculated.  Technically, this is the unix timestamp on January 1, 1970.<br/>
```

Comment:
Unix

"the millisecond of the day" is unclear to me.

Reply:
Updated the text.